webhook-connector/order-events.md

```diff
@@ -0,0 +1,131 @@
+---
+description: Order Events overview.
+---
+
+# Order Events
+
+When using the order webhook you may configure and receive below order events.
+
+## Order is issued or updated by buyer
+
+The buyer can issue new order lines or update existing issued lines.
+
+If confirmed delivery schedule or prices are updated, this will become a reopen request.
+
+| OrderEvent                  | Webhook Configuration |
+| --------------------------- |-----------------------|
+| `OrderIssuedByBuyer`        | New order is issued by the buyer |
+| `OrderLinesIssuedByBuyer`   | Additional order lines are added by the buyer |
+| `OrderLinesReissuedByBuyer` | Updated order lines are reissued by the buyer |
+| `OrderLinesUpdatedByBuyer`  | Order and line fields other than prices & delivery schedules are updated by the buyer |
+
+## Order confirmed by supplier or buyer
+
+The supplier can either accept, reject or propose an alternative (see next section).
+
+* Accepted means that the supplier confirmed the order line(s) as requested by the buyer.
```

Comment:
Nit: put the explanations in the table below.

Reply:
The text in the table are the literal texts in the webhook configuration you may choose.

Reply:
In the other tables it's a rephrasing / explanation of what such events mean. imo it makes sense to move these bullets to the table for consistency as well.